ReactWindows/ReactNative.Shared/Bridge/ReactContext.cs

```diff
@@ -326,7 +326,7 @@ public void HandleException(Exception exception)
         /// <remarks>
         /// This method should be called exactly once.
         /// </remarks>
-        internal void InitializeWithInstance(IReactInstance instance)
+        public void InitializeWithInstance(IReactInstance instance)
```

Comment:
You can add an `InternalsVisibleTo` attribute to the AssemblyInfo.cs file for ReactNative.Net46 for the time being.

---
In reply to: [83104991](https://github.com/ReactWindows/react-native-windows/pull/771#discussion_r83104991) [](ancestors = 83104991)